oldlog.go

```diff
@@ -97,6 +97,19 @@ func SetLogLevel(name, level string) error {
 	return nil
 }
 
+// GetSubsystems returns a slice containing the
+// names of the current loggers
+func GetSubsystems() []string {
+	subs := make([]string, len(loggers))
+
+	i := 0
+	for k := range loggers {
```

Comment:
Yeah that's the first solution that comes to mind, but I checked http://stackoverflow.com/questions/21362950/go-golang-getting-an-array-of-keys-from-a-map and in the end leaned towards the "efficient" solution, even it efficiency makes no point here, since both ways it is 100% clear what it does anyway.

I will change it if it makes you happy.